packages/playwright-core/src/server/injected/recorder/recorder.ts

```diff
@@ -1094,6 +1093,7 @@ export class Recorder {
     recreationInterval = this.injectedScript.builtinSetTimeout(recreate, 500);
     this._listeners.push(() => clearInterval(recreationInterval));
 
+    this.highlight.appendChild(createSvgElement(this.document, clipPaths));
```

Comment:
This change is a little scary to me. Let me check if the SVG icons are used anywhere outside the `highlight` element.

Reply:
All mentions of `url(#icon-...` are in `highlight.css`, so this should be fine.